frontend/docker-compose-e2e-tests.yml

```
@@ -6,7 +6,7 @@
 version: '3'
 services:
   db:
-    image: ghcr.io/cloudnative-pg/postgresql:15
+    image: ghcr.io/cloudnative-pg/postgresql:15.13-12
```

Definitely looks like a fix to someone else's bug though I don't like the minor version pinning. 🤔

Another option is to pull the official Postgres image from Docker Hub (suggested code). I also don't love it because it's prone to some latency and throttling from docker.io though I like it better than relying on a broken Docker image.

```suggestion
    image: docker.io/library/postgres:15-alpine
```

Can we please give that a try?

---

I have nothing against it. We can try